css/components/list.css

```diff
@@ -0,0 +1,61 @@
+@import url(../base/reset.css);
+@import url(../utilities/color.css);
+@import url(../components/avatar.css);
+@import url(../helpers/typography.css);
+
+/* styling for non-bulleted list and inline list*/
+.list-group,
+.list-group-bulleted {
+  padding-left: 1.5rem;
+}
+
+.list-group .list-item {
+  padding: 0.5rem 0;
+}
+
+.list-group-inline .list-item {
+  display: inline-block;
+  padding: 0.5rem;
```

Comment:
If px unit if the final value is <=10 px

Reply:
Done!